styles/main.css

```diff
@@ -44,103 +48,152 @@ a {
 .bio p {
     text-indent: 3em;
 }
+
 footer {
     font-size: 12px;
     text-align: center;
     margin-bottom: 2em;
 }
 
 header nav ul {
-    list-style: none; 
+    list-style: none;
     margin: 0;
 }
+
 ul {
     list-style: square inside;
 }
-.width-wrap{
+
+.width-wrap {
     max-width: 1024px;
     min-width: 300px;
     margin-left: auto;
     margin-right: auto;
     padding-right: 2em;
-    padding-left:2em;
+    padding-left: 2em;
     margin-bottom: 3em;
-    border-left:1px solid rgba(0, 0, 0, 0.3);
-    border-right:1px solid rgba(0, 0, 0, 0.3);
-    border-bottom:1px solid rgba(0, 0, 0, 0.3);
+    border-left: 1px solid rgba(0, 0, 0, 0.3);
+    border-right: 1px solid rgba(0, 0, 0, 0.3);
+    border-bottom: 1px solid rgba(0, 0, 0, 0.3);
     background-color: #f18973;
-    -moz-box-shadow: 4px 4px 4px rgba(255, 0 , 0, 0.3);
-    -webkit-box-shadow:  4px 4px 4px rgba(255, 0 , 0, 0.3);
-    box-shadow: 4px 4px 4px rgba(255, 0 , 0, 0.3);
-}
-.y {list-style: none; 
-    padding:0
-    }
- body{ padding-top: 0;
-       padding-bottom: 0;}
-.hhhh{padding:0}
-.mugshot img { box-shadow: 2px 2px 2px 3px rgba(0, 0, 0, .5);}
+    -moz-box-shadow: 4px 4px 4px rgba(255, 0, 0, 0.3);
+    -webkit-box-shadow: 4px 4px 4px rgba(255, 0, 0, 0.3);
+    box-shadow: 4px 4px 4px rgba(255, 0, 0, 0.3);
+}
+
+.y {
+    list-style: none;
+    padding: 0
+}
+
+body {
+    padding-top: 0;
+    padding-bottom: 0;
+}
+
+.hhhh {
+    padding: 0
+}
+
+.mugshot img {
+    box-shadow: 2px 2px 2px 3px rgba(0, 0, 0, .5);
+}
+
 footer hr {
-    width:50vw;
-    min-width:300px;
-    margin-top:3em;
-    margin-bottom:2em;
-    border-left:0;
-    border-right:0;
+    width: 50vw;
+    min-width: 300px;
+    margin-top: 3em;
+    margin-bottom: 2em;
+    border-left: 0;
+    border-right: 0;
     border-top: 3px solid #8b0000;
     border-bottom: 8px dotted #8b0000;
     border-top-right-radius: 8px;
     border-top-left-radius: 8px;
-    
-}
-*{box-sizing: border-box;
-}
- header div{background-image: url("/images/paper.png");

```

Comment:
This path is not correct.  You need to use `../` to select the parent directory of the stylesheet